samples/oauth_rails_vetverification/app/views/verification/_disability_rating.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="col-md-5">
+    <div class="card">
+        <ul class="list-group list-group-flush">
+            <li class="list-group-item"><strong>Decision</strong> <%= disability_rating['decision'] %></li>
```

Comment:
I like that this app exercises multiple endpoints.. It might be nice to visually show that the data for each comes from a separate endpoint -- with a link to the documentation for the endpoint and possibly even an expandable section that shows the json response (though that's a bit of a stretch)

Reply:
So to show it visually I added documentation links.  I also changed my VeteranVerification class to expose the response from the API and updated the readme to describe how to do that in rails console.  I thought added the JSON response into the page would clutter up my already ugly page, especially since I couldn't decide to show the raw string or try to bring in some other gem to make it look like a pretty hash or if I wanted to make new endpoints that show just the response.